package.json

```diff
@@ -1,71 +1,72 @@
-		"@fortawesome/free-regular-svg-icons": "^5.15.3",
-		"@fortawesome/free-solid-svg-icons": "^5.15.3",
-		"@fortawesome/react-fontawesome": "^0.1.14",
-		"@tailwindcss/line-clamp": "^0.2.1",
-		"@tippyjs/react": "^4.2.5",
-		"axios": "^0.21.1",
-		"classnames": "^2.3.1",
-		"framer-motion": "^4.1.17",
-		"next": "^11.0.1",
-		"next-images": "^1.8.1",
-		"next-seo": "^4.26.0",
-		"next-themes": "^0.0.15",
-		"react": "^17.0.2",
-		"react-dom": "^17.0.2",
-		"react-tilt": "^0.1.4",
-		"react-toastify": "^7.0.4",
-		"react-use": "^17.2.4",
-		"react-use-lanyard": "^0.0.8"
-	},
-	"devDependencies": {
-		"@types/node": "^15.12.5",
-		"@types/react": "^17.0.13",
-		"@typescript-eslint/eslint-plugin": "^4.28.1",
-		"@typescript-eslint/parser": "^4.28.1",
-		"autoprefixer": "^10.2.6",
-		"css-loader": "^5.2.6",
-		"eslint": "^7.29.0",
-		"eslint-config-next": "^11.0.1",
-		"eslint-plugin-react": "^7.24.0",
-		"eslint-plugin-react-hooks": "^4.2.0",
-		"onchange": "^7.1.0",
-		"postcss": "^8.3.5",
-		"prettier": "^2.3.2",
-		"sass": "^1.35.1",
-		"tailwindcss": "^2.2.4",
-		"typescript": "^4.3.5"
-	},
-	"browserslist": {
-		"production": [
-			">0.2%",
-			"not dead",
-			"not op_mini all"
-		],
-		"development": [
-			"last 1 chrome version",
-			"last 1 firefox version",
-			"last 1 safari version"
-		]
-	}
-}
+{
+	"name": "webpage",
+	"version": "4.0.0",
+	"description": "My personal webpage source >:3",
+	"author": {
+		"email": "demirci.baris38@gmail.com",
+		"name": "Barış DEMİRCİ",
+		"url": "https://barbarbar338.fly.dev"
+	},
+	"license": "GPL-3.0",
+	"scripts": {
+		"dev": "next",
+		"build": "next build",
+		"start": "next start --port 8080",
+		"lint": "next lint",
+		"format": "prettier --write .",
+		"format:watch": "onchange . -- prettier --write {{changed}}"
+	},
+	"dependencies": {
+		"@fortawesome/fontawesome-svg-core": "^1.2.35",
+		"@fortawesome/free-brands-svg-icons": "^5.15.3",
+		"@fortawesome/free-regular-svg-icons": "^5.15.3",
+		"@fortawesome/free-solid-svg-icons": "^5.15.3",
+		"@fortawesome/react-fontawesome": "^0.1.14",
+		"@tailwindcss/line-clamp": "^0.2.1",
+		"@tippyjs/react": "^4.2.5",
+		"axios": "^0.21.1",
+		"classnames": "^2.3.1",
+		"framer-motion": "^4.1.17",
+		"next": "^11.0.1",
+		"next-images": "^1.8.1",
+		"next-seo": "^4.26.0",
+		"next-themes": "^0.0.15",
+		"preact": "^10.5.14",
+		"react-tilt": "^0.1.4",
+		"react-toastify": "^7.0.4",
+		"react-use": "^17.2.4",
+		"react-use-lanyard": "^0.0.8"
+	},
+	"devDependencies": {
+		"@types/node": "^15.12.5",
+		"@types/react": "^17.0.13",
+		"@typescript-eslint/eslint-plugin": "^4.28.1",
+		"@typescript-eslint/parser": "^4.28.1",
+		"autoprefixer": "^10.2.6",
+		"css-loader": "^5.2.6",
+		"eslint": "^7.29.0",
+		"eslint-config-next": "^11.0.1",
+		"eslint-plugin-react": "^7.24.0",
+		"eslint-plugin-react-hooks": "^4.2.0",
+		"onchange": "^7.1.0",
+		"postcss": "^8.3.5",
+		"prettier": "^2.3.2",
+		"react": "^17.0.2",
```

Comment:
also we can remove react if we want to use preact

Reply:
I've moved it to devDependencies in case it might throw type errors or warnings

Reply:
Well then, let's not delete it for now.